src/wp2hugo/internal/wpparser/wp_parser_setup.go

```diff
@@ -543,13 +582,22 @@ func isTag(tag *rss.Category) bool {
 	return tag.Domain == "post_tag" || tag.Domain == "portfolio_tags" || tag.Domain == "product_tag"
 }
 
+func isTaxonomy(taxonomy *rss.Category, taxonomies []TaxonomyInfo) TaxonomyInfo {
```

Comment:
Nit: change this to `func isTaxonomy(taxonomy *rss.Category, taxonomies []TaxonomyInfo) *TaxonomyInfo {`